File: mainActivity/src/main/java/com/sierrawireless/avphone/task/IUserClient.java
package com.sierrawireless.avphone.task;

import java.util.List;

import net.airvantage.model.AirVantageException;
import net.airvantage.model.User;

public interface IUserClient {

    List<String> checkRights() throws AirVantageException;

    User getUser();
}

File: mainActivity/src/main/java/com/sierrawireless/avphone/task/IUserClient.kt
package com.sierrawireless.avphone.task

import net.airvantage.model.AirVantageException
import net.airvantage.model.User

interface IUserClient {

    val user: User?

    @Throws(AirVantageException::class)
    fun checkRights(): List<String>
}
